src/client/src/App.js

```diff
@@ -26,14 +25,26 @@ class App extends Component {
       searchResult: true,
     }
   }
+  
 
   render(){
       return (
         <BrowserRouter>
         <Switch>
           <Route exact path='/' render={() => (
             <div className="App">
-              
+
+              <div className='header'>
+                <div className="header-content">
+                  <h1 className="header-title" >Suffering</h1>
+                  <p >
+                    A FHIR based someting blah blah that gets you patient data
+                  </p>
+                  <a className="header-link" href="https://www.hl7.org/fhir/" target="_blank">About FHIR </a>
+                  <i class="fas fa-fire-alt"></i>
```

Comment:
I think these need to be `className` otherwise it throws an error in the console or is that just me?

Reply:
The class ones are from Bootstrap, so you might need to do an npm install

Reply:
I see, thanks!